train.py

```diff
@@ -137,7 +137,7 @@ def mask_data(data,args):
             if os.path.isdir(args.inputFile+infile): continue
             infile = os.path.join(args.inputFile,infile)
             if args.noHeader:
-                df_arr.append(pd.read_csv(infile, dtype=np.float64, header=0, nrows = args.nrowsPerFile, usecols=[*range(0, 48)], names=CALQ_COLS))
+                df_arr.append(pd.read_csv(infile, dtype=np.float64, header=0, nrows = args.nrowsPerFile, usecols=[*range(1,49)], names=CALQ_COLS))
```

Comment:
this is an interesting find, and can affect what we see with the electron training!